backend/src/api/attendees.js

```diff
@@ -0,0 +1,128 @@
+const express = require('express')
+const mongodb = require('mongodb')
+const router = express.Router()
+const { errorWrap } = require('../middleware')
+const { Attendee, Candidate } = require('../models')
+
+// get all attendees
+router.get(
+  '/',
+  errorWrap(async (req, res) => {
+    const attendees = await Attendee.find({})
+    res.json({
+      code: 200,
+      result: attendees,
+      success: true
+    })
+  })
+)
+
+// get one attendee
+router.get(
+  '/:netId',
+  errorWrap(async (req, res) => {
+    const netId = req.params.netId
+    const attendee = await Attendee.find({ netId })
+    res.json({
+      code: 200,
+      result: attendee,
+      success: true
+    })
+  })
+)
+
+// create a new attendee
+router.post(
+  '/',
+  errorWrap(async (req, res) => {
+    const data = req.body
+    const newAttendee = new Attendee({
+      name: data.name,
+      netId: data.netId,
+      email: data.netId + '@illinois.edu',
```

Comment:
hmmmmm the functionality I'm working on (Workspaces) is supposed to allow us to give the tool to other student groups and schools for them to use. So it's not necessarily true that the email should be suffixed with '@illinois.edu'. Similarly, the field 'netId' probably is not a great descriptor either (although this is a non-issue and more of a naming nit). I recommend dropping the email column entirely or having emails be an explicit entry.

Reply:
ok